src/app/css/_deviceDetail.scss

```diff
@@ -23,23 +23,16 @@
     margin-bottom: 15px;
 }
 
-.properties-section {
-    margin-top: 20px;
-    padding-bottom: 20px;
-    @include themify($themes) {
-        border: 1px solid themed('borderColor')
-    }
-    .properties-section-icon {
-        float: left;
-        margin-left: -5px;
+.properties-section-command-bar {
+    .ms-CommandBar {
+        border-bottom: 0px !important;
     }
-    .properties-section-command-bar {
-        .ms-CommandBar {
-            border-bottom: 0px !important;
-        }
 
-        .ms-OverflowSet-item{
-            padding-left: 25px;
-        }
+    .ms-OverflowSet-item{
+        padding-left: 25px;
     }
 }
+
+.sas-token-section {
+    margin: 0px 10px;
```

Comment:
Will likely need to add some css rules here for the spinner background color using themify.

Reply:
Can you provide more details on what you're asking? I don't think this component has a spinner.